Shopify/dawn
# Fix product review alignment when product doesn't have medias

### PR Summary: 


Fix the alignment for the product reviews when the product has no media. Applies to the main product page and the featured product section. 

### Why are these changes introduced?

Fixes alignment issues. Fixes https://github.com/Shopify/dawn/issues/3250

### What approach did you take?
I wrapped the content of the product rating block to be able to apply the alignment to it. 

### Visual impact on existing themes

The impact should be what the PR fixes. No regression on product with medias and center alignment when there are no medias for the product. 

### Testing steps/scenarios

Make sure to remove the white space as there has been some formatting changes done by the linter/theme check
![](https://screenshot.click/08-42-w2ct8-zf9o4.png)

- [ ] Check out the `Cyclamen` product for a media less one and make sure the review block is centered
- [ ] Same thing but add a featured product section with different product to make sure it works as expected. 
- [ ] One thing to watch out for is if we had any CSS selector that is impacted by the change in the DOM structure. 

### Demo links


- [Editor](https://admin.shopify.com/store/os2-demo/themes/162893758486/editor)

### Checklist
